packages/wrangler/src/deployment-bundle/esbuild-plugins/hybrid-nodejs-compat.ts

```diff
@@ -15,15 +14,24 @@ const REQUIRED_UNENV_ALIAS_NAMESPACE = "required-unenv-alias";
  * @param _unenvResolvePaths Root paths used to resolve absolute paths.
  * @returns ESBuild plugin
  */
-export function nodejsHybridPlugin(_unenvResolvePaths?: string[]): Plugin {
-	const { alias, inject, external } = env(nodeless, cloudflare);
+export async function nodejsHybridPlugin(
+	_unenvResolvePaths?: string[]
```

Comment:
These paths are no longer used, so we can remove it all the way to the top env vars.
But this also means that the current approach to testing in `packages/unenv-preset/tests/index.test.ts` isn't actually going to test against the local build.

Instead, I think it is time to move those checks (which is really an integration test with Wrangler) into the `fixtures/nodejs-hybrid-app` fixture alongside all the other tests that give the same coverage.

Then we can remove the `devDep` from this package to Wrangler, resolving the cyclic dependency, allowing  Wrangler to have a `workspace:*` dependency on this package.

Then we have clean simple dependency setup, the latest changes are automatically get tested and released together.

Reply:
Let's handle this in a follow up PR.

I think the unenv-preset should use miniflare for `packages/unenv-preset/tests/index.test.ts` because we do not want to wait for wrangler to have the updated workerd before testing - we want to be able to create a preset PR has soon as workerd is out (or even with a dev build).

As discussed before while we should be able to create the PR early, it should not be mergeable (i.e. tests failing) until whatever workerd version it relies on is deployed.

Does that sound good?

Reply:
And ideally we should keep the ability use those paths to have the ability to run tests a dev version of the preset.

For now this PR is using `require.resolve` to minimize the risk of having wrong resolved paths. But it would be great to switch to whatever resolution is used by unenv latest RC (it has changed recently) which allow overriding the base path.